translations/README.mx_spanish.md

```diff
@@ -0,0 +1,210 @@
+
+## Demos
+
+Revisa todos los ejemplos de aplicaciones en la carpeta de [community/sample-apps](../community/sample-apps).
+
+### Aplicaciones en tiempo real
+
+- Aplicación de Chat Grupal creada con React, incluye un indicador de escribiendo, usuarios en línea y notificaciones       para nuevos mensajes.
+  - [Pruébala](https://realtime-chat.demo.hasura.app/)
+  - [Tutorial](../community/sample-apps/realtime-chat)
+  - [Buscar APIs](https://realtime-chat.demo.hasura.app/console)
+ 
+- Aplicación de rastreo de ubicación en tiempo real, que muestra a un vehículo en movimiento cambiando sus coordenadas en   el GPS mientras se mueve en un mapa. 
+  - [Pruébala](https://realtime-location-tracking.demo.hasura.app/)
+  - [Tutorial](../community/sample-apps/realtime-location-tracking)
+  - [Buscar APIs](https://realtime-location-tracking.demo.hasura.app/console)
+
+- Un panel en tiempo real para agregar datos que estan cambio constante. 
+  - [Pruébala](https://realtime-poll.demo.hasura.app/)
+  - [Tutorial](../community/sample-apps/realtime-poll)
+  - [Buscar APIs](https://realtime-poll.demo.hasura.app/console)
+
+### Videos
+
+* [Agregar GraphQL a una instancia "auto-hosted" de GitLab](https://www.youtube.com/watch?v=a2AhxKqd82Q) (*3:44 mins*)
+* [Aplicación ToDo con Auth0 y servidor GraphQL](https://www.youtube.com/watch?v=15ITBYnccgc) (*4:00 mins*)
+* [GraphQL en GitLab integrado con aunteticación de GitLab](https://www.youtube.com/watch?v=m1ChRhRLq7o) (*4:05 mins*)
+* [Panel para 10 millones de viajes con geolocalización (PostGIS, Timescale)](https://www.youtube.com/watch?v=tsY573yyGWA) (*3:06 mins*)
+
+
+## Soporte y solución de problemas
+
+La documentación y la comunidad te ayudarán a solucionar la mayoría de los problemas. Si encuentras un error o necesitas contactarte con nosotros, puedes hacerlo usando uno de los siguientes canales: 
+
+* Soporte y retroalimentación: [Discord](https://discord.gg/vBPpJkS)
+* Problemas y seguimiento de errores: [GitHub issues](https://github.com/hasura/graphql-engine/issues)
+* Sigue las actualizaciones del producto: [@HasuraHQ](https://twitter.com/hasurahq)
+* Habla con nosotros en nuestro [chat](https://hasura.io)
+
+Estamos comprometidos a fomentar un ambiente abierto y acogedor en la comunidad. Por favor revisa el [Código de Conducta](../code-of-conduct.md).
+
+Si quieres reportar un problema de seguridad, por favor [lee esto](../SECURITY.md).
+
+## Contribuir
+
+Revisa nuestra [guía de contribución](../CONTRIBUTING.md) para más detalles.
+
+## Archivos de marca
+
+Los archivos de marca de Hasura (logotipos, la mascota de Hasura, la insignia: "powered by", etc.) pueden ser encontradas en la carpeta [assets/brand](../assets/brand). Siéntete libre de usarlos en tu
+aplicación, sitio web, etc. Estaremos encantados si agregas la insignia "Powered by Hasura" a tus aplicaciones creadas con Hasura. ❤️
+
+<div style="display: flex;">
```

Comment:
The Hasura logo doesn't show here:

![image](https://user-images.githubusercontent.com/5722022/69914039-e2df4680-143f-11ea-8d4c-45254856d457.png)

Reply:
done